docs/data/migration/migration-pickers-v7/migration-pickers-v7.md

```diff
@@ -312,6 +312,59 @@ const theme = createTheme({
 });
 ```
 
+### Clean the `ownerState`
+
+The `ownerState` is an object describing the current state of a given component to let you do more advanced customization.
+It is used in two different APIs:
+
+1. In the theme's `styleOverrides`, the `ownerState` is passed to the function that returns the style:
+
+```ts
+const theme = createTheme({
+  components: {
+    MuiDateCalendar: {
+      styleOverrides: {
+        root: ({ ownerState }) => ({
+          /** Style based on the ownerState */
+        }),
+      },
+    },
+  },
+});
+```
+
+2. In the `slotProps`, the `ownerState` is passed to the function that returns the custom props:
+
+```tsx
+<DatePicker
+  slotProps={{
+    actionBar: (ownerState) => ({
+      /** Props based on the ownerState */
+    }),
+  }}
+/>
+```
+
+Before version `v8.x`, the `ownerState` contained every prop of the component that was defining it, plus some additional internal states if needed.
```

Comment:
Do we need that specific clarification?
I don't see any additional clarity it brings... 🤔 🤷 
```suggestion
Before version `v8.x`, the `ownerState` contained every prop of the component, plus some additional internal states if needed.
```